packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -1199,6 +1204,18 @@ function PageRoot({ children }: PageRootProps) {
         {children}
       </Stack>
     </Container>
+  ) : (
+    <Stack
+      data-testid="page-root"
+      direction="column"
+      sx={{
```

Comment:
I see no vertical margin here, we should probably keep the `my: 2` from the Stack when there is a container?
No need to write things twice either, let's write the `<Stack` component only once  and make some of the styles change depending on the `container` setting.